### PR TITLE
[v50] Stac 15294 cli2 docs

### PR DESCRIPTION
**Pull request by gggina**

### CODE REVIEW

File: setup/cli/new-install.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+rm stackstate-cli.tar.gz
+
+# Step 3 - Verify installation success
+sts version
+```
+
+After installation the `sts` command is available to the current user from any path location.
+
+{% endtab %}
+{% tab title="Homebrew" %}
```
Comment (gggina) [inline setup/cli/new-install.md:77]: just `Coming soon!` or similar

File: setup/cli/new-install.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
+---
+description: StackState Self-hosted v4.6.x
+---
+
+# StackState CLI v2
+
+## Overview
+
+The StackState CLI provides easy access to the functionality provided by the StackState APIs. It can be used to configure StackState, work with data, automate using StackState data and help with developing and configuring StackPacks. You can configure the CLI to work with multiple instances of StackState.
```
[the remark this answers is not in the document]
Comment (lowdewijk) [reply setup/cli/new-install.md]: Done. See comparison page.

File: setup/cli/new-install.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+```bash
+sts cli save-config --url URL --api-token API-TOKEN 
+```
+
+## Uninstalling 
+
+{% tabs %}
+{% tab title="Windows" %}
+
+Open a **Powershell** terminal and execute each step one-by-one or all at once.
```
[the remark this answers is not in the document]
Comment (lowdewijk) [reply setup/cli/new-install.md]: done

File: setup/cli/old-install.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 description: StackState Self-hosted v4.6.x
 ---
 
-# StackState CLI
+# StackState CLI v1
 
 ## Overview
```
[the remark this answers is not in the document]
Comment (lowdewijk) [reply setup/cli/old-install.md]: done

**Pull request merged**